joudat_dir/README.md

```diff
@@ -0,0 +1,4 @@
+`How to run the file:`
+```python3 searcher.py```
+
+Enjoy!
```

Comment:
To the README you should add:
- The name of the project
- A list of contributors
- What it does
- Why you contributed it